incubator/java-openliberty/templates/kafka/README.md

```diff
@@ -0,0 +1,78 @@
+# Kafka template for Open Liberty
+
+This template can be used to develop Liberty applications that connect to Kafka by using MicroProfile Reactive messaging. A simple `StarterApplication` is included that enables basic production and consumption of events. 
+
+
+## Getting Started with the StarterApplication.
+
+### 1. Create a new folder and initialize it using appsody init:
+
+
+```
+mkdir test-appsody-kafka
+cd test-appsody-kafka
+appsody init java-openliberty kafka
+```
+
+### 2. Start Kafka and ZooKeeper
+
+In order to run the `StarterApplication` you must start Kafka and ZooKeeper containers. ZooKeeper is a dependency of Kafka.  Use the `docker-compose.yaml` that is provided in the template to start both containers. 
+
+
+Start docker compose with the following command:
+
+```docker-compose up```
+
+If you run `docker network list`, you should see a new network with the name of your project directory and the word `_default` appended. For example, `test-appsody-kafka_default`.
+
+Alternatively, if you want to connect to a Kafka broker elsewhere, edit `src/main/resources/META-INF/microprofile-config.properties` and set the value of the `mp.messaging.connector.liberty-kafka.bootstrap.servers` property to the host and port number of the your broker.
```

Comment:
Editing the **microprofile-config.properties** file would indeed take effect within the container.   Someone using this in "host mode" via `mvn liberty:dev` would need to either:
1. edit the pom.xml 
2. run like:  `mvn  -Dliberty.var.mp.messaging.connector.liberty-kafka.bootstrap.servers=localhost:9093`

Not sure if you want to complicate the doc here.. but FYI at least.

Reply:
For completeness..let me just note that editing `microprofile-config.properties` would also take effect for the prod image built by `appsody build`.     (If this needed to be distinct from the dev container value we could do profiles for that too).

Reply:
Hmm I think that would get a bit awkward to explain in the doc.  
For `appsody build` I think it should always be overridden in `app-deploy.yaml` with an env var anyway